Packaging.props

```diff
@@ -18,13 +18,19 @@
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
     <LineupPackageVersion Condition="'$(LineupPackageVersion)' == ''">2.0.0</LineupPackageVersion>
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">2.0.0</PlatformPackageVersion>
+    <PackageTargetRuntime Condition="'$(PackageTargetRuntime)' != '' AND '$(PackageTargetRuntimeSuffix)' != ''">$(PackageTargetRuntime)-$(PackageTargetRuntimeSuffix)</PackageTargetRuntime>
```

Comment:
I plan to move support for PackageTargetRuntimeSuffix into buildtools.  The purpose of this is to allow both TargetGroup and OSGroup to independently contribute to a compound RID.